From 79a9bf3743acb9ebaa22e6f8eaa5a817172cca78 Mon Sep 17 00:00:00 2001
From: Roman Mashak <mrv@mojatatu.com>
Date: Mon, 11 Dec 2017 16:24:31 -0500
Subject: [PATCH] ss: remove duplicate assignment

Fixes: 8250bc9ff4e5 ("ss: Unify inet sockets output")
Signed-off-by: Roman Mashak <mrv@mojatatu.com>
Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
(cherry picked from commit 9f1a9ae88862f0e5b1beef4bef582be0d8edf3d9)

Conflicts:
	misc/ss.c
-> Context changed due to missing commit 00ac78d39c29f
   ("ss: print tcpi_rcv_ssthresh").
---
 misc/ss.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/misc/ss.c b/misc/ss.c
index 45a0c330ec55f..cc1b930a19bcb 100644
--- a/misc/ss.c
+++ b/misc/ss.c
@@ -2227,7 +2227,6 @@ static void tcp_show_info(const struct nlmsghdr *nlh, struct inet_diag_msg *r,
 		s.sacked	 = info->tcpi_sacked;
 		s.fackets	 = info->tcpi_fackets;
 		s.reordering	 = info->tcpi_reordering;
-		s.rcv_space	 = info->tcpi_rcv_space;
 		s.cwnd		 = info->tcpi_snd_cwnd;
 
 		if (info->tcpi_snd_ssthresh < 0xFFFF)
-- 
2.13.1

